### PR TITLE
feat: Modal Component

### PR DESCRIPTION
**Pull request by ozairkhan2810**
Added Modal Component
Simple static Modal Page

image

### CODE REVIEW

File: documentation/index.html [hunk 1]
```diff
@@ -36,7 +36,7 @@
   <!-- content -->
   <div class="flex">
     <!-- sidebar content -->
-    <!-- <input class="sidebar-toggle" type="checkbox"> -->
+    <!-- <input class="sidebar-toggle" type="checkbox"> --> mb-1
```
Comment (sks-21) [inline documentation/index.html]: remove unnecessary commented code

File: documentation/index.html [hunk 2]
```diff
@@ -130,9 +131,7 @@ <h3>Badge</h3>
       <div id="button" class="ml-1 newline pb-1 mb-1">
         <h3>Button</h3>
         <p>
-          Buttons are the most common and most important part of a web app
-          that allows you to navigate from one page to another, perform
-          certain actions etc.
+          Buttons are the most common and most important part of a website that allows you to navigate from one page to another, perform certain actions etc.
```
Comment (sks-21) [inline documentation/index.html]: use code formatter

File: documentation/index.html [hunk 3]
```diff
@@ -175,101 +174,96 @@ <h3>Button</h3>
...
         <p class="text-content">Text only card</p>
-        <div class="card-text-only">
-          <div class="card-text-only-main">
-            <h3 class="card-title">IPL - Indian Premier League</h3>
-            <h5 class="card-author">by Ozair Khan</h5>
+        <div class="card-text-only mb-1">
+          <div class="card-body">
+            <div class="card-content">IPL</div>
+            <div class="card-sub-content">The Indian Premier League is a professional men's Twenty20 cricket league,
+              contested by ten teams based out of ten Indian cities.</div>
```
Comment (sks-21) [inline documentation/index.html]: Same as above

**Review by sks-21 — APPROVED** [review]
Suggested some changes. Rest good to merge.

**Review by shaksham08 — APPROVED** [review]
Looks good to me

**Pull request merged**